Feat: Support component apis and update online.yaml

madfrdanskebank opened the pull request:
Add support for multiple APIs that can be automatically merged.

This should help those who would like to have an automatic setup to sync their generated open api documents without having to manually update the file files.

madfrdanskebank committed:
Feat: Support component apis

Add support for multiple APIs that can be automatically merged.

This should help those who would like to have an automatic setup to sync their generated open api documents without having to manually update the file files.

madfrdanskebank committed:
review fixes: correct package versions

madfrdanskebank committed:
Update merge-all.sh

madfrdanskebank committed:
try to fix execution permissions for github actions

edvinasmuncelis commented on openapi/online.yaml:
```diff
@@ -1399,6 +1427,47 @@ paths:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/SimulateRequestByAliasRequest'
+          text/json:
```
text/json, application/*+json do you need these ?

edvinasmuncelis commented on component-apis/online/regression-tester.yaml:
```diff
@@ -0,0 +1,61 @@
+          application/json:
+            schema:
+              $ref: '#/components/schemas/SimulateRequestByAliasRequest'
+          text/json:
```
same here and check if there are more occurrences

madfrdanskebank committed:
Update online apis

rename regression-tester to cardpassthorugh-regression-tester to match bc name, and update both apis to only include application/json content types unless otherwise specified.
